### ISSUE TITLE
CSV with BOM are not working

### ISSUE THREAD

**Issue by tchapi** [parsecsv/parsecsv-for-php#204]
With: 

[test_with_BOM.csv](https://github.com/parsecsv/parsecsv-for-php/files/6590830/test_with_BOM.csv)

```php
$csv = new Csv();
$csv->delimiter = ';';
$csv->parseFile('test_with_BOM.csv');
var_dump($csv->data);
```

The result is : 

```
array(0) {
}
```

For information:

```
❯ file /test_with_BOM.csv
/test_with_BOM.csv: UTF-8 Unicode (with BOM) text, with CRLF line terminators
```

This may be related to https://github.com/parsecsv/parsecsv-for-php/issues/193

If I'm doing something that is not correct, please do not hesitate to tell me (the CSV and the code seem pretty standard though)

**Comment by tchapi** [comment]
Aaaaah I've just stumbled on https://github.com/parsecsv/parsecsv-for-php/pull/202
I think this is the culprit

**Comment by Fonata** [comment]
Indeed. I have now released [version 1.3.1](https://github.com/parsecsv/parsecsv-for-php/releases/tag/1.3.1), which should fix this issue.
